app.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var userName = prompt('What should I call you?');
+  alert('Okay, ' + userName + ', my name is Sean.');
+
+var userAck = prompt('Before you READ about me, let\'s play a game to get to know each other, shall we?');
+  if (userAck.toLowerCase() === 'yes' || userAck.toLowerCase() === 'y') {
+  alert('Let\'s do this!');
+} else if (userAck.toLowerCase() === 'no' || userAck.toLowerCase () === 'n') {
+  alert('Okay, fine, be that way...but I\'m still going to bother you with questions.');
+} else if (userAck !==true){
+  alert('Okaaaaay, let\'s move on');
+}
+var playerReady = confirm ('Are you ready to play "The Guessing Game?"');
+  if (playerReady == true) {
+  alert('Okay! Let\'s Go!')
+} else {
+  alert('Okay, you\'re starting to piss me off. You can just read about me then. Have fun with that! Okay, just joking, let\'s move on');
+}
+var userAge = prompt ('How old do you think I am?');
+  if (userAge > 33) {
+  alert('Not sure if I should take that as a compliment or...')
+  } else {
+  alert('Good guess! But I\'m a little older.');
+}
+var myHairColor = prompt ('Is my hair brown?');
+if (myHairColor.toLowerCase() === 'yes' || myHairColor.toLowerCase() === 'y') {
+alert('Great job, ' +userName);
```

Comment:
this needs to be indented, as it is nested inside the if statement.  this also applies to lines 30, 32, 36, 38, 40, 44, 46, 48, 52, 54, 56, 60, 62, 64